### PR TITLE
refactor: minor pass through twap imrovements

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

Some minor pass through changes discovered while reviewing #2168 
## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not applicable

### CODE REVIEW

File: x/gamm/twap/store.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (k Keeper) storeNewRecord(ctx sdk.Context, twap types.TwapRecord) {
 // returns the TWAP record from state for (id, t', asset0, asset1),
 // where t' is such that:
 // * t' <= t
-// * there exists no t'' in state, where t'' < t'
+// * there exists no t'' in state, where t' < t''
```
Comment (p0mvn) [inline x/gamm/twap/store.go:98]: Wasn't 100% sure on this but it seems to be the case

File: x/gamm/twap/store.go [hunk 2]
```diff
@@ -140,6 +140,6 @@ func (k Keeper) getRecordAtOrBeforeTime(ctx sdk.Context, poolId uint64, t time.T
 			return twap, nil
 		}
 	}
-	return types.TwapRecord{}, fmt.Errorf("TWAP not found, but there are twaps available for this time."+
-		" Were provided asset0denom and asset1denom (%s, %s) correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
+	return types.TwapRecord{}, fmt.Errorf("TWAP not found, but there are other twaps available for this time."+
+		" The provided asset0denom and asset1denom (%s, %s) are correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
```
Comment (p0mvn) [inline x/gamm/twap/store.go:144]:
```suggestion
		" Were provided asset0denom and asset1denom (%s, %s) are correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
```

File: x/gamm/twap/store.go [hunk 3]
```diff
@@ -140,6 +140,6 @@ func (k Keeper) getRecordAtOrBeforeTime(ctx sdk.Context, poolId uint64, t time.T
 			return twap, nil
 		}
 	}
-	return types.TwapRecord{}, fmt.Errorf("TWAP not found, but there are twaps available for this time."+
-		" Were provided asset0denom and asset1denom (%s, %s) correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
+	return types.TwapRecord{}, fmt.Errorf("TWAP not found, but there are other twaps available for this time."+
+		" Were provided asset0denom and asset1denom (%s, %s) are correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
```
Comment (p0mvn) [inline x/gamm/twap/store.go:144]:
```suggestion
		" Were provided asset0denom and asset1denom (%s, %s) correct, and in order (asset0 > asset1)?", asset0Denom, asset1Denom)
```

File: x/gamm/twap/store.go (hunk 1 above)
Comment (ValarDragon) [reply x/gamm/twap/store.go:98]: The old comment is more directionally correct, but still has a bug. It should read:
```suggestion
// * there exists no `t'' <= t` in state, where `t' < t''`
```

We return `t' < t` and the guarantee is there is no other state record with a time greater than t' but less than t (namely that t' is the closest to t, disallowing entries > t)

File: x/gamm/twap/keeper.go [hunk 4]
```diff
@@ -23,5 +23,5 @@ func NewKeeper(storeKey sdk.StoreKey, transientKey *sdk.TransientStoreKey, param
 
 // TODO: make this read from a parameter, or hardcode it.
 func (k *Keeper) PruneEpochIdentifier(ctx sdk.Context) string {
-	return "daily"
+	return "day"
```
Comment (alexanderbez) [inline x/gamm/twap/keeper.go:26]: we should make these constants :)

**Review by alexanderbez — APPROVED** [review]

File: x/gamm/twap/keeper.go (hunk 4 above)
Comment (p0mvn) [reply x/gamm/twap/keeper.go]: Also made this to address the TODO: https://github.com/osmosis-labs/osmosis/issues/2289

**Pull request merged**